src/jit/gentree.cpp

```diff
@@ -16592,6 +16594,16 @@ CORINFO_CLASS_HANDLE Compiler::gtGetClassHandle(GenTree* tree, bool* isExact, bo
                     objClass              = lvaTable[objLcl].lvClassHnd;
                     *isExact              = lvaTable[objLcl].lvClassIsExact;
                 }
+                else if (base->OperGet() == GT_ARR_ELEM)
```

Comment:
Indentation of the else is off.

Reply:
Hmm, looks like it's a bug in GitHub diff viewer. It's not showing several lines that I see in the raw file.